setup.py

```diff
@@ -60,6 +60,118 @@ def generate_proto(source):
 generate_proto("jax/experimental/australis/executable.proto")
 generate_proto("jax/experimental/australis/petri.proto")
 
+extras_require = {
```

Comment:
FYI I didn't change the `extras_require` dict at all, just moved it. The real changes start at L163.